src/client/lib/hooks/useRecaptchaTestUtils.ts

```diff
@@ -0,0 +1,47 @@
+// Used to mock the `load` and `error` events for scripts so the @guardian/libs `loadScript` method functions as intended.
```

Comment:
this file is related to the test right?

it would be worth adding to a `__tests__` directory inside the `hooks` folder to separate it from the non-test code. I also noticed there's a couple of other files in the `hooks` folder too which should also go into the `__tests__` folder

Reply:
will do!

Reply:
moved all the relevant files into a `__tests__` folder